packages/css/src/logo/README.md

```diff
@@ -0,0 +1,35 @@
+# Logo
+
+Het logo van de gemeente bestaat uit een beeldmerk (3 Andreaskruisen) en een woordmerk (Gemeente Amsterdam).
+Het logo heeft een vaste kleur, grootte en positie en staat op al onze websites.
+
+## Specificaties
+
+Gebruik altijd het online logo.
```

Comment:
Aren't all the logo's we offer 'online'? Maybe this should be something like:

```suggestion
Gebruik altijd het algemene 'Amsterdam'-logo.
```

Reply:
These docs are only relevant to the Design System, right? The non-'online' logo's are irrelevant to these docs imo.

Reply:
I agree. Let’s add this context then and remove the ambiguity of what the ‘online’ or ‘general Amsterdam’ logo actually is.

```suggestion
Het design system biedt het correcte logo voor online aan.
Er bestaan andere varianten van het logo, maar die mogen op websites en applicaties niet gebruikt worden.
```

Reply:
Small tweak:

```suggestion
Het design system biedt het correcte logo voor online gebruik aan.
Er bestaan andere varianten van het logo, maar die mogen op websites en in applicaties niet gebruikt worden.
```